docker/from-packages/scripts/start-collabora-online.sh

```diff
@@ -36,8 +36,7 @@ fi
 
 # Replace trusted host and set admin username and password - only if they are set
 if test -n "${domain}"; then
```

Comment:
I guess we should actually turn this into a 'if test -n "${domain}" -o -n "${aliasgroup1}"; then' ... so that it is not executed twice when there is both domain and aliasgroupX in there?  Or better - ignore the domain when there is an aliasgroupX, but warn that both are provided, and the aliasgroupX takes the precedence?